lib/school.rb

```diff
@@ -0,0 +1,20 @@
+class School
+  def initialize(start_time, hours_in_school_day)
+  end
+
+  def start_time
+    @start_time = '9:00'
+  end
+
+  def hours_in_school_day
+    @hours_in_school_day = 7
+  end
+
+  def student_names
+    @student_names = []
+  end
+
+  def add_student_name(name)
+    @name = []
```

Comment:
Here, you want to shovel the name argument into the @student_names instance variable:

`@student_names << name`